src/service-metrics/index.ts

```diff
@@ -3,7 +3,7 @@
 
```

Comment:
any reason for renaming this file?

Reply:
I'll change this back. As you said in the larger comment I got caught up in the refactoring and ended up slipping things in I shouldn't have. Sorry about that and thanks for bearing with me!